src/application/extension-popup-menu/app.tsx

```diff
@@ -0,0 +1,55 @@
+import { Route, Routes } from 'react-router-dom';
+
+import { useExtensionSettings } from 'shared/extension';
+import { Closable } from 'shared/ui';
+
+import { Footer } from './footer';
+import { PopupHomeView } from './popup-home-view';
+import { TopBar } from './top-bar';
+import { CustomizationSettings } from './settings-view/customization-settings';
+import { MainSettingsMenu } from './settings-view/main-settings-menu';
+
+export const EXTENSION_POPUP_MENU_ROUTES = {
```

Comment:
feels like `MENU` is unnecessary UI detail, `EXTENSION_POPUP_ROUTE` feels good enough, also note that I removed (s) to make it as a singular, it could be ROUTE(S) if it was an array but here you will be referring to a single route once consuming it: `EXTENSION_POPUP_ROUTE.HOME`

Reply:
also I feel like `menu` can be removed from the module name `src/application/extension-popup` is good enough since there is only 1 extension popup